src/core/Akka.Remote.Tests.Performance/InboundMessageDispatcherSpec.cs

```diff
@@ -32,9 +32,11 @@ private class BenchmarkActorRef : MinimalActorRef
         {
             private readonly Counter _counter;
 
-            public BenchmarkActorRef(Counter counter)
+            public BenchmarkActorRef(Counter counter, RemoteActorRefProvider provider)
```

Comment:
:+1: 
(we should really go through all of our code to find `==` on actorrefs, I do think we have a few of those. but that is for another issue/task)